### DISCUSSION TITLE
Referecing an element from array in .yml

### DISCUSSION THREAD

**Discussion by ViliusSutkus89** [community/community#25712] [Actions]
<p>Hello,</p>
<p>I’m trying to get commit message in “on push” workflow.</p>
<p>I assumed I could just get it by using something like</p>
<pre><code>- run: echo ${{ github.event.push.commits.0.message }}
</code></pre>
<p>Sadly, I can’t, because .0 to get first item from array is invalid syntax.</p>
<p>How can I do this?</p>

**Comment by wipe2238** [comment, marked as answer]
<pre><code>- run: echo "${{ github.event.commits[0].message }}"
</code></pre>

**Comment by giyama** [comment]
<p>Helped me a lot here! Thank you!</p>
<p>If somebody is looking for labels:</p>
<p>Print the whole array:</p>
<ul>
<li>run: echo ‘${{ toJSON(github.event.pull_request.labels) }}’</li>
</ul>
<p>Print as list:</p>
<ul>
<li>run: echo “${{ toJson(github.event.pull_request.labels.*.name) }}”</li>
</ul>
<p>Print item:</p>
<ul>
<li>run: echo “${{ toJson(github.event.pull_request.labels[0].name) }}”</li>
</ul>
